src/lib/checksum.ts

```diff
@@ -2,7 +2,8 @@ import ReactNativeBlobUtil from 'react-native-blob-util';
 import { btoa } from 'abab';
 
 const checksum = async ({ path }: { path: string }) => {
-  const md5 = await ReactNativeBlobUtil.fs.hash(path, 'md5');
+  const decodedPath = decodeURIComponent(path);
```

Comment:
if the encoding was done using encodeURI, then we'll prolly want decodeURI actually.